nvm.sh

```diff
@@ -1109,7 +1109,7 @@ nvm_compute_checksum() {
     command gsha256sum "${FILE}" | command awk '{print $1}'
   elif nvm_has "openssl" && ! nvm_is_alias "openssl"; then
     nvm_err 'Computing checksum with openssl dgst -sha256'
-    command openssl dgst -sha256 "${FILE}" | rev | command awk '{print $1}' | rev
+    command openssl dgst -sha256 "${FILE}" | command awk '{print $NF}'
```

Comment:
`echo abc | rev | awk '{print $1}'` prints out `cba` - `echo abc | awk '{print $NF}'` prints out `abc`. These are not equivalent.